python/tvm/relay/op/transform.py

```diff
@@ -20,6 +20,7 @@
 
 from . import _make
 from ..expr import TupleWrapper, const
+from ...tir.expr import IntImm
```

Comment:
Probably better change to `from ...tir import expr as _expr`. And probably don't need the fix for the doc then.

Reply:
good idea. changed.